### ISSUE TITLE
Instance search inhibits holes for instance fields

### ISSUE THREAD

**Issue by identicalsnowflake** [agda/agda#2488]
```agda
postulate
  Class : Set

record I : Set where
  field
    {{c}} : Class

postulate
  class : Class

i : I
i = record {
    c = ?
  }
```

If you load the above, Agda will throw an error saying it couldn't find an instance for `c`--which is true except that it shouldn't be looking for one in the first place! A hole in `c` should indicate that I intend to solve this problem myself, so it shouldn't stop me from trying by saying it can't find a solution.

**Comment by identicalsnowflake** [comment]
Not sure when exactly this was fixed, but it works correctly now. It no longer gives an unwanted error preventing you from working on the hole.

**Comment by andreasabel** [comment]
This was most probably fixed by the unfixing of #2172.
